dashboard/src/components/dashboard.svelte

```diff
@@ -39,17 +42,17 @@
 	$: message = 'No devices found';
 	$: query, onSearch();
 	$: tableData = {
-		headings: ['Device Id', 'Related channel'],
+		headings: ['Device Name', 'Related channel'],
 		rows: searchResults.map((device) => ({
 			onClick: () => handleSelectDevice(device),
 			content: [
 				{
 					icon: 'person-badge',
 					boxColor: 'transparent',
-					value: device.id ?? '-'
+					value: device.name ?? '-'
 				},
 				{
-					value: device.channelAddress ?? '-'
+					value: `channel-${device.name}` ?? '-'
```

Comment:
In reality this information don't need to be inferred. What happens if the backend logic in giving the names will change? This information need to be provided by the backend together with the name.